### PR TITLE
Render bevy error codes

### PR DESCRIPTION
**Pull request by NiklasEi**
This PR generates a list of error codes from the markdown files in the bevy repository. They are rendered at `/errors`.

- [x] wait for bevyengine/bevy#2994

### CODE REVIEW

**Review by NiklasEi — COMMENTED** [review]
~I will try pulling common parts of the two generation crates (assets/errors) into some kind of utility.~ (Edit: this doesn't really seem worth it at the moment)

Just wanted to have the draft PR open already so we have a place to discuss where/how to show the errors on the website in general.

**Comment by alice-i-cecile** [comment]
Seems like a sensible idea. I like this.

**Comment by mockersf** [comment]
for the website, which documents the released version, this will need to pull the `latest` branch of Bevy

**Comment by NiklasEi** [comment]
> for the website, which documents the released version, this will need to pull the `latest` branch of Bevy

Good point. The codes now come from `latest`. To still be able to merge this before the `0.6` release, I added some general information on the page and link to the descriptions for Bevy main. If it's fine that the page is pretty empty until the release, we could merge this before `0.6`.

File: templates/learn.html [hunk 1]
```diff
@@ -55,5 +55,18 @@
       </div>
     </div>
   </a>
+  <a class="card" href="/errors">
+    <div class="card-image">
+      <img src="/assets/bevy_icon_dark.svg" class="centered-card-image" alt="Bevy logo" />
+    </div>
+    <div class="card-text">
+      <div class="card-title">
+        Bevy Error index
```
Comment (mockersf) [inline templates/learn.html:64]:
```suggestion
        Bevy Error Index
```

**Comment by mockersf** [comment]
It would be nice to have "Bevy Assets" and "Bevy Error Index" each have their own custom logo, but that can be done later. Something like the Bevy logo and a file for assets, and the logo with a warning sign for errors?

**Review by mockersf — APPROVED** [review]

**Comment by cart** [comment]
Yeah I'm down to merge but I think we should remove Bevy Error Index from the `/learn` page until we have useful content here and we have a unique logo for the error index (i dont take issue with reusing the bevy logo as much as I do with the fact that two identical images back to back cuts down on perceived quality).

File: templates/learn.html [hunk 2]
```diff
@@ -55,5 +55,18 @@
       </div>
     </div>
   </a>
+  <a class="card" href="/errors">
```
Comment (cart) [inline templates/learn.html:58]: I'm thinking errors should probably live under the `/learn` url

Comment (cart) [reply templates/learn.html:58]: `/learn/errors`

**Pull request merged**